spack/packages/dla-future/package.py

```diff
@@ -46,22 +47,13 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
         description="Build C API compatible with ScaLAPACK",
     )
 
-    variant(
-        "mpi-gpu-aware",
-        default=False,
-        when="@master",
-        description="Use GPU-aware MPI.",
-    )
-    conflicts(
-        "+mpi-gpu-aware",
-        when="~cuda ~rocm",
-        msg="GPU-aware MPI requires +cuda or +rocm",
-    )
+    variant("mpi-gpu-aware", default=False, when="@0.5.0:", description="Use GPU-aware MPI.")
+    conflicts("+mpi-gpu-aware", when="~cuda ~rocm", msg="GPU-aware MPI requires +cuda or +rocm")
 
     variant(
         "mpi-gpu-force-contiguous",
         default=True,
-        when="@master +mpi-gpu-aware",
+        when="@0.5.0: +mpi-gpu-aware",
         description="Force communication buffers to be contiguous before communicating.",
```

Comment:
Noticed only on the spack PR (https://github.com/spack/spack/pull/44463#discussion_r1622445995), but maybe we can change it here as well to:
```suggestion
        description="Force GPU communication buffers to be contiguous before communicating.",
```
?